examples/Org.OData.Capabilities.V1.capabilities.md

```diff
@@ -31,8 +31,8 @@ The item level has no named entity set. It is annotated using `NavigationRestric
         "NonInsertableProperties": ["Items/uuid"]
       },
       "UpdateRestrictions": {
-     // "Updatable": {"$Path": "Items/canUpdate"},
-     // "NonUpdatableProperties": ["Items/uuid"],
+        "DEPRECATED-Updatable": {"$Path": "Items/canUpdate"},
+        "DEPRECATED-NonUpdatableProperties": ["Items/uuid"],
```

Comment:
```suggestion
     // "Updatable": {"$Path": "Items/canUpdate"},
     // "NonUpdatableProperties": ["Items/uuid"],
```

Reply:
That leads to ugly coloring after I switched on JSON syntax, because it is considered an error.